CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## TBD
+### Changed
+- `MessageReader.read()` now returns `MessageEvent`, wrapping `ListenerEvent` and `MessageProperty`.
+
+### Added
+- `SqsMessageReader.read()` no longer deletes messages. Instead, `SqsMessageReader.delete()` and `SqsMessageReader.extendVisibilityTimeout()` have been added to give more control over inflight messages.
```

Comment:
`extendVisibilityTimeout()` is that valid where is it?

Reply:
Ah yes, that was silly, this must go

Thanks